src/Okta.Sdk/Generated/IlogClientInfo.Generated.cs

```diff
@@ -5,10 +5,6 @@
 
 // This file was automatically generated. Don't modify it directly.
```

Comment:
Do you know why the file name case is wrong? `IlogClientInfo` instead of `ILogClientInfo`
(the class name is correct)